plugins/ansible-collection-theforeman-foreman/debian/rules

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 
 %:
-	dh $@
+	dh $@ --buildsystem=none
```

Comment:
we now ship the Makefile in the collection, which makes `dh` try to use the `Makefile` buildsystem, and that breaks all the things.